test/iidm/ThreeWindingsTransformerTest.cpp

```diff
@@ -155,6 +157,78 @@ Network createThreeWindingsTransformerTestNetwork() {
      return network;
 }
 
+void addPhaseTapChangerLeg2(ThreeWindingsTransformer& transformer, Terminal& terminal) {
+    transformer.getLeg2().newPhaseTapChanger()
+        .setTapPosition(2L)
+        .setLowTapPosition(1L)
+        .beginStep()
+        .setAlpha(9.0)
+        .setB(10.0)
+        .setG(11.0)
+        .setR(12.0)
+        .setRho(13.0)
+        .setX(14.0)
+        .endStep()
+        .beginStep()
+        .setAlpha(14.5)
+        .setB(15.0)
+        .setG(16.0)
+        .setR(17.0)
+        .setRho(18.0)
+        .setX(19.0)
+        .endStep()
+        .beginStep()
+        .setAlpha(20.5)
+        .setB(20.0)
+        .setG(21.0)
+        .setR(22.0)
+        .setRho(23.0)
+        .setX(24.0)
+        .endStep()
+        .setRegulationMode(PhaseTapChanger::RegulationMode::ACTIVE_POWER_CONTROL)
+        .setRegulating(true)
+        .setRegulationTerminal(stdcxx::ref<Terminal>(terminal))
+        .setRegulationValue(25.0)
+        .setTargetDeadband(0.0)
+        .add();
+}
+
+void addPhaseTapChangerLeg3(ThreeWindingsTransformer& transformer, Terminal& terminal) {
+    transformer.getLeg3().newPhaseTapChanger()
+        .setTapPosition(-2L)
+        .setLowTapPosition(-3L)
+        .beginStep()
+        .setAlpha(-9.0)
+        .setB(-10.0)
+        .setG(-11.0)
+        .setR(-12.0)
+        .setRho(-13.0)
+        .setX(-14.0)
+        .endStep()
+        .beginStep()
+        .setAlpha(-14.5)
+        .setB(-15.0)
+        .setG(-16.0)
+        .setR(-17.0)
+        .setRho(-18.0)
+        .setX(-19.0)
+        .endStep()
+        .beginStep()
+        .setAlpha(-20.5)
+        .setB(-20.0)
+        .setG(-21.0)
+        .setR(-22.0)
+        .setRho(-23.0)
+        .setX(-24.0)
+        .endStep()
+        .setRegulationMode(PhaseTapChanger::RegulationMode::ACTIVE_POWER_CONTROL)
+        .setRegulating(false)
+        .setRegulationTerminal(stdcxx::ref<Terminal>(terminal))
```

Comment:
Is it possible to do not use the explicit cast?

Reply:
No it isn't because reference_wrapper constructor is explicit
```
    explicit reference_wrapper(T& reference) noexcept :
        m_pointer(std::addressof(reference)) {
    }
```